bin/nib-ls

```diff
@@ -214,18 +218,23 @@ def proc_file(f, opts):
         else:
             row += ['error']
 
-    if opts.stats:
+    if opts.stats or opts.counts:
         # We are doomed to load data
         try:
             d = vol.get_data()
             if not opts.stats_zeros:
                 d = d[np.nonzero(d)]
-            # just # of elements
-            row += ["[%d] " % np.prod(d.shape)]
-            # stats
-            row += [len(d) and '%.2g:%.2g' % (np.min(d), np.max(d)) or '-']
+            if opts.stats:
+                # just # of elements
+                row += ["@l[%d]" % np.prod(d.shape)]
+                # stats
+                row += [len(d) and '@l[%.2g, %.2g]' % (np.min(d), np.max(d)) or '-']
+            if opts.counts:
+                items, inv = np.unique(d, return_inverse=True)
```

Comment:
you really don't want me off the hook, don't you? ;)  Let's may be chat about something more interesting at some point ;)

ok -- will do now for len "properly" ;)